subjects/podiumposition/README.md

```diff
@@ -0,0 +1,41 @@
+## podiumposition
+
+### Instructions
+
+A F1 race just finished and the commentator is calling the finishing positions incorrectly.
+Help to fix this before the contestants arrive at the podium by providing the commentator with the correct podium position.
+
+Write a function `PodiumPosition` that takes an array of arrays of type `string` and returns the competitor positions correctly.
```

Comment:
Does it take an array of arrays or a slice of slices? arrays or slices? https://go.dev/tour/moretypes/6

Reply:
Thank you Esteban for noticing it, it is a slice of slices and it's fixed!